jsonnet/rules.libsonnet

```diff
@@ -465,10 +465,6 @@ function(params) {
     {
       name: 'openshift-ingress.rules',
       rules: [
-        {
-          expr: 'sum by (code) (rate(haproxy_server_http_responses_total[5m]) > 0)',
```

Comment:
Is there a replacement for this impemented in any of the networking edge components?